src/constants.ts

```diff
@@ -20,9 +20,6 @@ export const BASE_URL = process.env.BASE_URL || ITERABLE_API_URL;
 
 export const GETMESSAGES_PATH = '/inApp/web/getMessages';
 
-/** @todo update once new endpoint is ready */
-export const CACHE_ENABLED_GETMESSAGES_PATH = '/newEndpoint';
```

Comment:
In `/inapp/request.ts`, line 25, would you mind updating the commented out line to read "tbd" or something to replace the reference to the placeholder constant?